src/site/apt/examples/deploy-ssh-external.apt

```diff
@@ -58,9 +58,26 @@ Deployment of artifacts in an external SSH command
 
 +----+
 
- If you are deploying from Unix or have Cygwin installed you won't need to any additional configuration in your <<<settings.xml>>>
- file as everything will be taken from the environment. But if you are on Windows and are using something like <<<plink>>> then
- you will need something like the following:
+ If you are deploying from Linux/Unix or have Cygwin installed you only need to add the location of your PEM based private key in your <<<settings.xml>>>
+ 
++----+
+
+<settings>
+  ...
+  <servers>
+    <server>
+      <id>production-server</id>
```

Comment:
`id` should be the same as in `distributionManagement`

Reply:
thanks, consistency is improved now.